<!DOCTYPE html>
<html>
  <head>
    <title>2019_APTF_nomapview.utf8</title>
    <meta charset="utf-8">
    <link rel="stylesheet" href="peek_default.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">




background-image: url(images/IMGP1696-MYLF.jpg)
background-size: cover
background-position: 50% 75%
class: center, inverse, title-slide

.font90[# Hybridization between two sympatric ranid frog species in the northern Sierra Nevada]

&lt;html&gt;
  &lt;div style='float:left'&gt;&lt;/div&gt;
  &lt;hr color='#DAAA00' size=1px width=720px&gt;
&lt;/html&gt;

&lt;br&gt;
&lt;br&gt;
&lt;br&gt;

### Ryan A. Peek, Mallory Bedwell, Sean O’Rourke, Greta M. Wengert, &lt;br&gt; Caren Goldberg, Michael R. Miller

&lt;br&gt;

![:scale 70%](images/CWS_logo_small.png)
]

???

- As freshwater systems and biota are rapidly declining globally, conservation efforts will require assessment of the adaptive capacity of populations to rapid environmental change.

- Small populations with limited genetic diversity may have reduced adaptive potential and difficulty responding to future environmental change.

---
class: center, inverse

## Acknowledgements

&lt;html&gt;
  &lt;div style='float:left'&gt;&lt;/div&gt;
  &lt;hr color='#DAAA00' size=1px width=720px&gt;
&lt;/html&gt;

.pull-left[

- ### .font70[Center for Watershed Sciences &amp; colleagues]

- ### .font70[Corey Luna, Rick Wachs, and Sarah Mussulman]

- ### .font70[Thanks to Jamie Bettaso, Mourad Gabriel, Cathy Brown, Colin Dillingham, Tina Hopkins, Neil Keung, and WSU and USFS field crew members]

]

.pull-right[

![](images/mcloud_psre_2006_IMGP0277.jpg)
]

---
background-image: url(https://photos.smugmug.com/Nature/Landscapes/Via-Ryan/Panoramas/i-cpNnd8s/0/4d5b3bff/XL/IMG_0709-XL.jpg)
background-position: 10% 0%
class: inverse, bottom, left

## Hybridization

--

 - ### .font80[Hybridization may be an important evolutionary mechanism (homogenization, speciation, adaptation)]

--

 - ### .font80[Hybridization between closely related taxa can promote gene flow (introgression), or combine parental genotypes in ways that yield reproductively sterile or isolated lineages]

--

 - ### .font80[Hybridization events may be short-lived and difficult to detect, and are generally considered rare in vertebrates]

???
- Cross-breeding between species, or hybridization, in vertebrates has generally been considered a rare occurrence. 
- Hybridization may be an important evolutionary mechanism for either homogenization (reversing initial divergence between species), speciation (from reproductive isolation of hybrid populations), or adaptation (transfer of adaptive alleles) (Abbott et al., 2013; Barrera-Guzmán, Aleixo, Shawkey, &amp; Weir, 2018; Mallet, 2007).

---
class: inverse, center, middle

# Can *Rana sierrae* (RASI) and &lt;br&gt; *R. boylii* (RABO) hybridize?

&lt;html&gt;
  &lt;div style='float:left'&gt;&lt;/div&gt;
  &lt;hr color='#DAAA00' size=1px width=720px&gt;
&lt;/html&gt;

.pull-left[
&lt;img src="images/IMGP1696-MYLF.jpg" height="220" width="300"&gt;
*Sierra yellow-legged frogs (RASI)*
]

.pull-right[
&lt;img src="images/rabo_A_P1060730.jpg" height="220" width="300"&gt;
*Foothill yellow-legged frogs (RABO)*
]

--

**Probably not based on Zweifel's work (1955)**

???
 - Unlike other ranid frog species with broad areas of potential intergradation (Shaffer et al 2004) RASI/RABO rarely occur sympatrically
 - breeding experiments by Zweifel (1955) found high incidences of embryological abnormalities—indicating a post-zygotic barrier between the species (low viability in fertilization). 
 - However, these experiments only crossed female R. sierrae with male R. boylii from very different sites (Butte and Nevada County vs. Contra Costa County). 

---
class: inverse, top, left

## Management Challenges:

 - ### Species have very similar morphology &amp; habitat preferences Zweifel (1955)
 
--

 - ### Detecting hybrids &amp; assigning individuals to species in overlapping zone can be difficult, and imprecise using field identification methods

.pull-right[

![:scale 75%](images/P1060762.JPG)

]

---
background-image: url(images/IMG_9096.JPG)
background-size: 400px
background-position: 110% 95%
class: inverse, left, top

## Objectives:

&lt;html&gt;
  &lt;div style='float:left'&gt;&lt;/div&gt;
  &lt;hr color='#DAAA00' size=1px width=720px&gt;
&lt;/html&gt;

--

.pull-left[

- ### .font80[Use RADSeq/Rapture to look for hybridization and identify species]

- ### .font80[Estimate migration rates between species across 3 watersheds]

- ### .font80[Revise "map" to help USFS with management/conservation objectives]

]

???

- Hybridization can promote gene flow (introgression) between species
- Important evolutionary mechanism for: 
   - homogenization, adaptation, or speciation



---
class: inverse, left, top

# Foothill yellow-legged frogs (*Rana boylii*)

&lt;html&gt;
  &lt;div style='float:left'&gt;&lt;/div&gt;
  &lt;hr color='#DAAA00' size=1px width=720px&gt;
&lt;/html&gt;

--

.pull-left[

- ### Obligate river breeding frog, uses wide range of habitat (&lt;1500m), but has disappeared from over 50% of historical range


- ### Being evaluated as candidate for state and federal listing under ESA

]

.pull-right[

![](images/rabo_range_poly_FEA.png)

]

---
class: inverse, left, top

# Sierra yellow-legged frogs (*Rana sierrae*)

&lt;html&gt;
  &lt;div style='float:left'&gt;&lt;/div&gt;
  &lt;hr color='#DAAA00' size=1px width=720px&gt;
&lt;/html&gt;

--

.pull-left[

- ### Federally listed species, can use wide range of lotic/lentic habitat (~1,500 - 3,660m)

- ### Northern boundary of the species range is largely in the Feather basin

]

.pull-right[

![](images/sylf_range_iucn_FEA.png)

]

---
class: left, top, inverse

# Methods: RADSeq &amp; RAD-Capture (RAPTURE)

&lt;html&gt;
  &lt;div style='float:left'&gt;&lt;/div&gt;
  &lt;hr color='#DAAA00' size=1px width=720px&gt;
&lt;/html&gt;

 - ### genome-wide, targeted genotyping approach (*Ali et al. 2016*)

![:scale 50%](images/ali_cover.png)

???
Flexible and cost efficient, thousands of samples multiplexed at once (high throughput).

- Many amphibian species cryptic/lack data, Large genome size, assessment and monitoring hard to do
- *restriction site-associated DNA sequencing*, highly efficient genotyping across thousands of individual samples for targeted loci
- For frogs, can sample with non-invasive methods using mouth swabs, or tadpole tail clips
- (i.e., small pops. with limited connectivity, low diversity)
- *RAPTURE/RADSeq* is a powerful and effective method

RAD Tags = sequences downstream of restriction sites. In order for samples to be sequenced on an Illumina machine, after cutting fragments must be ligated to adapters that will bind to an Illumina flow cell (RADSeq uses modified Illumina adapters that enable the binding and amplification of restriction site fragments only)

---
background-image: url(images/rapture_slides_1.jpg)
background-size: cover
class: center

???
 - Digest with Sbf1
 - Add RAD adapters
 - Sonicate/shear
 - purify
 - specify Capture baits/loci of interest (better/more sequencing at locations you want to look at)
 - Different cutters at different frequencies in the genome, this is rougly 300 cut sites per 100,000 bp

PCR duplicates can be identified as fragments that are identical in insert length and sequence composition, because random shearing ensures that fragments at a given locus are unlikely to be of equal length unless they are duplicates (Fig. 1; Davey et al. 2011; Hohenlohe et al. 2013). In contrast, for methods without a random-shearing step, all fragments at a given locus are expected to be of equal length whether or not they are PCR duplicates.  Pooled DNA was sheared in a Bioruptor NGS sonicator.

---
class: center, top, inverse

## Frog Samples &amp; Sequencing

&lt;html&gt;
  &lt;div style='float:left'&gt;&lt;/div&gt;
  &lt;hr color='#DAAA00' size=1px width=720px&gt;
&lt;/html&gt;

![](images/flare_helix.png)


.div-left[
- .font130[**1,443 individual Rana frogs from 17 species**]
]

--

.div-left[
- .font130[Designed ~8,500 Rapture baits (“tags”)]
 ]

--

.div-left[
- .font130[Paired end sequence data from Illumina HiSeq]
]

--

.div-left[
- .font130[*de novo* assembly via Novoalign and PRICE assembler, ANGSD for analysis]
]

---
background-image: url(images/legos_dna.png)
background-size: 300px
background-position: 100% 95%
class: inverse, left, top

### Sequencing Results

.div-left[
- .font100[Samples collected across 56 different localities in 3 watersheds]
]

--

.div-left[
- .font100[Filtered &amp; retained 311 samples that contained &gt;25,000 alignments, with mean alignments per sample = 229,485]
 ]

--

.div-left[
- .font100[Mean number of samples per site = 8]
]

--

.div-left[
- .font100[Kept all sites regardless of samples/site]
]

---
background-image: url(images/figure_01_overview_hybrid_300dpi.png)
background-size: 900px
class: inverse, left, bottom

### Hybridization Sampling Sites

???

Bean Creek/Spanish Creek largely the only locations where both species have been documented together, however, additional samples from Slate Creek and a few other small tributaries may show both spp (Goldberg/Mallory's MS)

---
class: inverse, center, top

## All but 2 individuals were assigned to species using PCA

--

![](images/figure_02_pca_rasi_all_25k_combined.png)
### Strong pattern of divergence between the species

---
background-image: url(images/figure_03_admix_rana_by_watershed_25k_k2_scaled.png)
background-size: 900px
background-position: 40% 100%
class: left, top, inverse

## Admixture

--

.pull-right[

![](images/hybrid_individual.png)
]

???
 - Used NGSAdmix to assess population structure and individual ancestry from genome-wide SNPs
 - Admixture showed the same two unknown samples from the Feather basin were had approximately 50% ancestry from each species.
 - ancestry in the individuals designated as “unknown” in the field also showed very low levels of mixed ancestry between the species
 - very low or nearly non-existent levels of mixed-ancestry in the American and Yuba basins as compared to the Feather. However, introgression between spp appears asymmetric, with a greater proportion of ancestry from R. boylii occurring in the R. sierrae samples

---
class: inverse, left, top

## F1 vs. F2 Test on hybrids

- ### Tested whether the hybrids were F1 (first-generation) or F2 (progeny of two F1’s)

--

- ### Identified putative species diagnostic SNPs (n=3,062)

--

- ### Expect **F1** hybrids to be heterozygous at all SNPs, 

--

- ### Expect **F2** hybrids to be heterozygous for at 50% of SNPs and homozygous at the remaining 50% with 25% assigned to each species

---
background-image: url(images/figure_04_f1_f2_hybrid_test_faceted.png)
background-size: 680px
class: top, left

.font80[### F1 vs. F2 Test]

???
 - Our filtering process (see Methods) yielded 3,062 putative diagnostic SNPs that were homozygous for different alleles in R. sierrae or R. boylii samples and also had successfully called genotypes in the two hybrid individuals

 - **We observed extremely high heterozygosity and very low homozygosity (6% genotyped as R. boylii, 4% R. sierrae, and 89% were heterozygous) (Figure 4). This level of heterozygosity is far greater than expected for F2 individuals**

- In addition, both hybrid individuals had *RASI* mitochondrial DNA (Bedwell 2018), indicating the maternal ancestor was from a *RASI* individual and the paternal ancestor was *RABO*
 
  - non Hz SNPs may be expected (at low levels), due to low coverage sequencing data; genotyping from low coverage sequencing will cause a low frequency of erroneous homozygous calls, because only one of the two alleles is sampled, causing heterozygotes to be called as homozygotes

---
background-image: url(images/figure_04b_hybrid_bean_ck_only_annot.png)
background-size: 800px
class: left, top

&lt;br&gt;
&lt;br&gt;
.font80[### Hybrid Locations]

---
class: inverse, left, top

## Demographic Modeling 

--

- ### Used **`fastsimcoal2`** (Excoffier &amp; Foll, 2011) coalescent simulations to estimate migration rates between species within each watershed

--

- ### Used all individuals except the two hybrids to calculate migration probability (or the per generation likelihood that any gene from one population transfers to another)

---
background-image: url(images/figure_05_coalescent_ancMigration_ptsCI.png)
background-size: 450px
background-position: 5% 40%
class: inverse, left, top

## Demographic Modeling 

--

.pull-right[

.font75[- ### Migration rates highest in Feather compared to Yuba and American]
]

--

.pull-right[

.font75[- ### Divergence time estimated at 371,000 generations or ~742 kya to 1.1 mya if using generation time of 2-3 years]
]

???

The best model based on maximum likelihood estimated the time since divergence between R. sierrae and R. boylii was 370,856 (370,041–371,670) generations.

Early Pleistocene began  ~1.8 mya ("Great Ice Age").


---
class: inverse, left, top

# Summary:

&lt;html&gt;
  &lt;div style='float:left'&gt;&lt;/div&gt;
  &lt;hr color='#DAAA00' size=1px width=720px&gt;
&lt;/html&gt;

### Strong divergence &amp; limited hybridization detected between RASI and RABO, however F1 hybrids were identified in Bean Creek

--

### Demographic modeling suggests migration rates between species are highest in the Feather Basin, and divergence estimates coincide with onset of Pleistocene Ice Age

--

### Hybridization outcomes that fail to produce successful offspring (sterile F1 hybrids) may have a greater cost on the species with low numbers of effective breeders (i.e., RASI), something to consider under future scenarios (climate change, population crash, etc)

???

 - ### unlikely that there is currently the potential for major introgression between R. sierrae and R. boylii, particularly as hybridization initially may not be adaptive and is often selected against (Abbott et al., 2013; Streicher et al., 2014).
 
 - ### hybridization outcomes that fail to produce successful offspring (sterile F1 hybrids) may have a greater cost on the species with low numbers of effective breeders, affecting both locally adapted populations and negatively impacting the probability of population persistence in a given region 
 
 - ### under future scenarios (e.g., warming climate, range contraction, population crashes) the loss of even several breeding individuals (via reproduction with R. boylii) may have a significant impact in declining populations
---
background-image: url(images/g_w_froglet.jpg)
background-size: 450px
background-position: 100% 15%
class: inverse, left, middle

&lt;br&gt;

.pull-left[
.font110["*The good life of any river may depend on the perception of its music; and the preservation of some music to perceive.* 
**Aldo Leopold**"]

![](images/flare_helix.png)

]

---
background-image: url(images/IMGP0800.jpg)
background-size: cover
class: inverse, center

## Questions?
    </textarea>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script src="macros.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function() {
  var d = document, s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})();</script>

<script>
(function() {
  var i, text, code, codes = document.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
})();
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
